package/libpiksi/libpiksi/src/version.c

```diff
@@ -53,6 +53,8 @@ int version_parse_str(const char *str, piksi_version_t *ver)
 {
   size_t len = strlen(str);
   size_t idx = 0;
+  char tail[len];

```

Comment:
Can we use inline init here? Also, confused about how this compiles, I thought array sizes needed to be constants?